pkg/v14/resource/tiller/resource.go

```diff
@@ -65,9 +73,30 @@ func (r *Resource) Name() string {
 	return Name
 }
 
-func (r *Resource) ensureTillerInstalled(ctx context.Context, clusterGuestConfig v1alpha1.ClusterGuestConfig) error {
+func (r *Resource) ensureTillerInstalled(ctx context.Context, obj interface{}) error {
+	objectMeta, err := r.toClusterObjectMetaFunc(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// Tenant Tiller is not deleted so cancel the resource. Tiller deployment
+	// will be deleted when the tenant cluster resources are deleted.
```

Comment:
How are the resources are deleted though? Will everything just vanish with the deleted cluster? Is that why we do not want to bother here? When this is the case the comment could be a little bit clearer. And I know it is difficult to come up with something self explanatory, but the log below is also rather 50 50. 🤷‍♂️

Reply:
Good to raise this. For me it's a question of responsibilities.

cluster-operator is responsible for updating all resources in the control plane. Otherwise we will have leftovers there.

Tiller is running in the tenant cluster and it's the responsibility of aws-operator etc to ensure all tenant cluster resources are deleted. We could delete tiller in the tenant cluster but it would lose part of its control plane.

I've updated the comment and the log message. WDYT?

Reply:
I would go with two logs. 

```
not deleting tiller in tenant cluster
tiller in tenant cluster will be deleted with cluster deletion. 
```

Something like that. Otherwise let's go with the version you had before.

Reply:
That's nice thx! I've gone with that.